pkgs/by-name/ne/nettee/package.nix

```
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  cmake,
+  versionCheckHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nettee";
+  version = "0.3.5";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/nettee/nettee-${finalAttrs.version}.tar.gz";
+    hash = "sha256-WeZ18CLexdWy8RlHNh0Oo/6KXxzShZT8/xklAWyB8ss=";
+  };
+
+  patches = [ ./fix-cmake-output.patch ];
+
+  nativeBuildInputs = [ cmake ];
+
```

```suggestion
  # additional shell scripts require accudate (not in nixpkgs)
  postInstall = ''
    rm $out/bin/*.sh
  '';
```
No point in having these scripts then, I would think.

---

OK, done